aries-site/src/components/content/PageBackground.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { Box, Image, ResponsiveContext } from 'grommet';
+import { useDarkMode } from '../../utils';
+
+export const PageBackground = ({ backgroundImage }) => {
+  const size = useContext(ResponsiveContext);
+  const themeMode = useDarkMode().value ? 'dark' : 'light';
+
+  return (
+    <Box
+      margin={
+        backgroundImage[size] && backgroundImage[size].margin
+          ? backgroundImage[size].margin
+          : backgroundImage.margin
+      }
+      pad={
+        backgroundImage[size] && backgroundImage[size].pad
+          ? backgroundImage[size].pad
+          : backgroundImage.pad
+      }
+      responsive={false}
+      width={size !== 'small' ? '65%' : 'fill'}
+    >
+      <Image
+        src={backgroundImage.src[themeMode]}
+        alt={backgroundImage.alt}
+        fit={backgroundImage.fit}
+        fill={backgroundImage.fill}
```

Comment:
we can deconstruct these variables on line 9 and use them directly.

Reply:
Done.